wgpu-core/src/instance.rs

```diff
@@ -772,6 +838,42 @@ impl Global {
         Ok(id)
     }
 
+    /// Creates a new surface from the given drm configuration.
+    ///
+    /// # Safety
+    ///
+    /// - All parameters must point to valid DRM values.
+    ///
+    /// # Platform Support
+    ///
+    /// This function is only available on Unix-like platforms (Linux, FreeBSD) and
```

Comment:
```suggestion
    /// This function is only available on non-apple Unix-like platforms (Linux, FreeBSD) and
```

(suggestion, take-or-leave): Maybe this (and the others) would be more clear, as Mac is a Unix-like as well